src/KeyworkerProfile/containers/KeyworkerProfileContainer.js

```diff
@@ -147,10 +147,10 @@ class KeyworkerProfileContainer extends Component {
             },
           }
         )
+        await this.getKeyworkerAllocations()
         setMessageDispatch('Offender allocation updated.')
         keyworkerChangeListDispatch([])
       }
-      history.push('/')
```

Comment:
I have a slight concern that removing this makes it inconsistent with other pages. But I can't understand why this or many of the other `history.push` calls are present, so I am not going to expect it to be kept in.

Reply:
as per slack this moves it to the empty page, so needs to go